providers/openstack/scs/cluster-class/values.yaml

```diff
@@ -1,15 +1,12 @@
-# mirrored from variables.tf
 controller_flavor: SCS-2V-4-20s
-worker_flavor: SCS-2V-4-20
-restrict_kubeapi: []
-# newly introduced:
-openstack_loadbalancer_apiserver: false
-# TBD, currently needed:
+identityRef:
+  cloudName: openstack
+  name: openstack
 images:
   controlPlane:
-    name: ubuntu-capi-image-v1.32.1
+    name: ubuntu-capi-image-v1.32.5
   worker:
-    name: ubuntu-capi-image-v1.32.1
-identityRef:
-  name: openstack
-  cloudName: openstack
+    name: ubuntu-capi-image-v1.32.5
+openstack_loadbalancer_apiserver: false
```

Comment:
This changes the default to not have a LB in front of the kubernetes-apiserver. Why?
(Since I switched all my defaults to octavia-ovn, I have started to consider LBs as cheap and reliable resource ...)